### PR TITLE
Explicitly say to open files in code editor

### PR DESCRIPTION
**Pull request by jhodgdon-drp**
See #1042

### CODE REVIEW

File: en/css/README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ It was written by programmers who worked for Twitter. Now it's developed by volu
 
 ## Install Bootstrap
 
-To install Bootstrap, you need to add this to your `<head>` in your `.html` file:
+To install Bootstrap, you need to add this to your `<head>` in your `.html` file (in the code editor):
```
[the remark this answers is not in the document]
Comment (jhodgdon-drp) [reply en/css/README.md]: Good idea! I'll edit that line.

File: en/dynamic_data_in_templates/README.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ So now we want published blog posts sorted by `published_date`, right? We alread
 Post.objects.filter(published_date__lte=timezone.now()).order_by('published_date')
 ```
 
-Now we put this piece of code inside the `blog/views.py` file by adding it to the function `def post_list(request)`, but don't forget to first add `from django.utils import timezone`:
+Now we put this piece of code inside the `blog/views.py` file (open it in the code editor) by adding it to the function `def post_list(request)`, but don't forget to first add `from django.utils import timezone`:
```
Comment (ekohl) [inline en/dynamic_data_in_templates/README.md]: This could also be phrased as `... file using the code editor. Add it to ..` for a bit smoother sentence.

Comment (jhodgdon-drp) [reply en/dynamic_data_in_templates/README.md]: OK, I'll revise. Thanks!

Comment (jhodgdon-drp) [reply en/dynamic_data_in_templates/README.md]: Revised in next commit.

**Comment by ekohl** [comment]
Thanks again :)

**Pull request merged**